app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,18 @@
+class CommentsController < ApplicationController
+  def create
+    user = User.find(params[:user_id])
+    post = user.posts.find(params[:post_id])
+    values = params.require(:comment).permit(:text)

```

Comment:
- [OPTIONAL] Let's make it a good practice to use strong parameters to better handle params according to [this lesson](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md#:~:text=Pay%20special%20attention%20to%20the%20strong%20parameters%20description%20in%20Rails%20Guides)
```ruby
    private 
    def comment_params
      params.require(:comment).permit(:text)
    end
```